[core]/es_extended/server/classes/player.lua

```diff
@@ -642,7 +642,7 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 			if _type ~= "string" then
 				return print(("[^1ERROR^7] xPlayer.setMeta ^5value^7 should be ^5string^7 as a subIndex!"):format(value))
 			end
-
+            self.metadata[index] = table.create(0, 0)
```

Comment:
Why create new table? So it will always be overwritten if there is some data already under the key.

This is not a good solution, what should be done here is to check if metadata[index] exists, if not then create an empty table for it but only if it doesn't exist.

For example: 

```lua
-- data mean index but this is example

if not self.metadata['data'] then
      self.metadata['data'] = {}
end

self.metadata['data']['newKey'] = "any value"

```